sdk/tables/aztable/connection_string.go

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package aztable
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+var ErrConnectionString = errors.New("connection string is either blank or malformed")
```

Comment:
I find it helpful when the error message includes an example of the expected format, in case the user is just totally lost on what you're asking them for:

<for instance, you have this one like this in your code - I'd clean this one up so it doesn't say 'dummy' though>
"DefaultEndpointsProtocol=http;AccountName=dummyaccount;AccountKey=secretkeykey;EndpointSuffix=core.windows.net"